src/Lists/PhoneNumberSettingsListBuilder.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\sms\Lists\PhoneNumberSettingsListBuilder.
+ */
+
+namespace Drupal\sms\Lists;
+
+use Drupal\Core\Config\Entity\ConfigEntityListBuilder;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Drupal\sms\Provider\PhoneNumberProviderInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Builds a list of phone number settings.
+ */
+class PhoneNumberSettingsListBuilder extends ConfigEntityListBuilder {
+
```

Comment:
this assertion is incorrect. verification information is only supplementary

it would be best to take a look at the list on a live site...

https://www.drupal.org/node/2643692#comment-10830266